src/client/Microsoft.Identity.Client/Internal/Requests/DeviceCodeRequest.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.Identity.Client.ApiConfig.Parameters;
 using Microsoft.Identity.Client.Core;
 using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.PlatformsCommon.Shared;
```

Comment:
Are these still needed (done several places)?

Reply:
Not needed. removing